src/components/CreateUser.js

```diff
@@ -133,156 +139,95 @@ class CreateUser extends React.Component {
 
-  renderButton = () => {
-    const buttonStyle = {
-      border: "0px",
-      borderRadius: "5px",
-      fontFamily: "'Josefin Slab', sans-serif",
-      fontSize: "1.3rem",
-      color: "#dddcdf",
-      backgroundColor: "#857e8f",
-      height: "40px",
-    };
+  renderLoader = () => (
+    <div className="login-form-loader">
+      <RingLoader color={"#857e8f"} size={50} loading={this.state.waiting} />
+    </div>
+  );
 
-    return (
-      <div style={{ alignSelf: "center", margin: "auto", paddingBottom: "10px" }}>
-        <button style={buttonStyle} type="submit">
-          Create Account
-        </button>
-      </div>
-    );
-  };
+  renderButton = () => (
+    <button className="login-form-button" style={{ width: "37.5%" }} type="submit">
+      Create Account
+    </button>
+  );
+
+  renderLink = () => (
+    <Link to="/login" className="login-form-link">
+      Already have an account? Click here to login
+    </Link>
+  );
 
   renderModal = () => {
-    const modal = {
-      container: {
-        fontFamily: "'Josefin Slab', sans-serif" /*font for the whole create page*/,
-        marginTop: "2%" /*modal distance from top of the screen*/,
-        marginLeft: "5%" /*modal distance from left side of the screen*/,
-        width: "40%",
-      },
-      form: {
-        display: "flex",
-        flexDirection: "column",
-        alignItems: "flex-start",
-        justifyContent: "center",
-        padding: "15px",
-        color: "#dddcdf",
-        backgroundColor: "#272134",
-      },
-      header: { fontSize: "3em", fontWeight: "bold", marginBottom: "10px" },
-    };
     return (
-      <div className="create-modal" style={modal.container}>
-        <form style={modal.form} onSubmit={this.submit}>
-          {this.renderHeader(modal)}
-          <div
-            style={{
-              width: "70%",
-              display: "flex",
-              flexDirection: "column",
-              justifyContent: "flex-start",
-              alignItems: "flex-start",
-            }}
-          >
-            {this.renderInputs()}
-            <div
-              style={{
-                display: "flex",
-                flexDirection: "column",
-                alignSelf: "center",
-                justifyContent: "flex-start",
-                alignItems: "center",
-              }}
-            >
-              <RingLoader color={"#171124"} size={50} loading={this.state.waiting} />
-              {this.state.errorMessage ? (
-                <div style={{ color: "red" }}>{this.state.errorMessage}</div>
-              ) : (
-                <span />
-              )}
-              {this.renderButton()}
-              <Link to="/login" className="create-form-link">
-                Already have an account? Click here to log in
-              </Link>
-            </div>
-          </div>
+      <div className="login-modal">
+        <form className="login-form" onSubmit={this.submit}>
+          {" "}
```

Comment:
Is there a reason that this is here? Is it the modal hack thing? @connorbo97

Reply:
Yup, modal hack. It's probably a good idea to investigate why its needed/remove it. I'll make a card in trello bout it.